knowledge-base/grid-customize-hierarchy-expand-column-blazor-grid.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Customize Hierarchy Expand Column in Blazor Grid
+description: Learn how to customize the hierarchy expand/collapse column in Telerik Blazor Grid, including changing its position, locking it, and setting its width and title.
+type: how-to
+page_title: How to Customize the Hierarchy Expand/Collapse Column in Telerik Blazor Grid
```

Comment:
Same as for the `title`. The page title appears in search results, so if it's specific, it's immediately clear if the reader should click it.